src/Invictus.Testing/LogicAppNotUpdatedException.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Runtime.Serialization;
+using System.Security.Permissions;
+
+namespace Invictus.Testing
+{
+    /// <summary>
+    /// Thrown when the logic app running in Azure could not be updated.
+    /// </summary>
+    [Serializable]
+    public class LogicAppNotUpdatedException : LogicAppException
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LogicAppNotUpdatedException"/> class.
+        /// </summary>
+        public LogicAppNotUpdatedException()
```

Comment:
I would remove this overload, we need to have logic app information as a minimum

Reply:
See ⏫ .